src/migrations/00091-drop-device-should-be-running-release-column.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "device"
+DROP COLUMN IF EXISTS "should be running-release";
+DROP INDEX IF EXISTS "device_should_be_running_release_application_idx";
```

Comment:
I think that we do not really need this, since dropping the column will drop the index as well.